src/components/views/settings/DevicesPanel.js

```diff
@@ -177,13 +188,14 @@ export default class DevicesPanel extends React.Component {
         );
     }
 
-    _renderDevice(device) {
+    _renderDevice(device, editable) {
         const DevicesPanelEntry = sdk.getComponent('settings.DevicesPanelEntry');
         return <DevicesPanelEntry
             key={device.device_id}
             device={device}
             selected={this.state.selectedDevices.includes(device.device_id)}
             onDeviceToggled={this._onDeviceSelectionToggled}
+            editable = {editable}
```

Comment:
please match the style of the surrounding code:
```suggestion
            editable={editable}
```